assopy/tests/test_admin.py

```diff
@@ -120,3 +120,13 @@ def test_user_new_order_admin(admin_client):
     response = admin_client.get(url)
 
     assert response.status_code == 200
+
+
+@mark.django_db
+def test_user_send_verification_email(admin_client):
+    assopy_user = factories.AssopyUserFactory()
+    url = reverse('admin:auser-send-verification-email', kwargs={'uid': assopy_user.user.id})
+
+    response = admin_client.get(url)
+
+    assert response.status_code == 302
```

Comment:
Could you also add a test that an email does get sent (https://docs.djangoproject.com/en/1.11/topics/testing/tools/#email-services) ? And that when the user is active, it doesn't?